typings/atom-ide/index.d.ts

```diff
@@ -178,7 +178,7 @@ export interface CodeActionProvider {
     editor: TextEditor,
     range: Range,
     diagnostics: Diagnostic[],
-  ): Promise<CodeAction[]>;
+  ): Promise<CodeAction[] | null>;
```

Comment:
The implementation of `getCodeActions` in `AutoLanguageClient` can return null, so had to add it here